### ISSUE TITLE
Question about compartmentalized ciliogenesis vs cytosolic ciliogenesis

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-ontology#22199]
Hi @krchristie  - think that you are the best person to answer this :-)

I see that we have the term cytosolic ciliogenesis in the GO but not compartmentalized ciliogenesis.

I was looking at this paper https://pubmed.ncbi.nlm.nih.gov/34357392/ where "Twitchy is required for compartmentalised and cytosolic ciliogenesis" and I wasn't sure if 'non-motile cilium assembly' GO:1905515 was equivalent to compartmentalized ciliogenesis?

**Comment by krchristie** [comment]
Hi @hattrill 

The simple answer is that 'non-motile cilium assembly' GO:1905515 is **NOT** _**equivalent**_ to compartmentalized ciliogenesis, though **ALL** 'non-motile cilium assembly' GO:1905515 **IS** compartmentalized ciliogenesis, according to Avidor-Reiss & Leroux 2015 (abstract copied in below). However, **MOST** 'motile cilium assembly is also compartmentalized ciliogenesis. 

Avidor-Reiss T, Leroux MR. **Shared and Distinct Mechanisms of Compartmentalized and Cytosolic Ciliogenesis.** Curr Biol. 2015 Dec 7;25(23):R1143-50. doi: 10.1016/j.cub.2015.11.001. PMID:26654377 https://pubmed-ncbi-nlm-nih-gov.ezproxy.jax.org/26654377/

> **Most motile** and **all non-motile** (also known as primary) eukaryotic cilia possess microtubule-based axonemes that **are assembled at the cell surface to form hair-like or more elaborate compartments** endowed with motility and/or signaling functions. **Such compartmentalized ciliogenesis depends on the core intraflagellar transport (IFT) machinery and the associated Bardet-Biedl syndrome complex (BBSome)** for dynamic delivery of ciliary components. **The transition zone (TZ), an ultrastructurally complex barrier or ‘gate’ at the base of cilia, also contributes to the formation of compartmentalized cilia**. Yet, **some ciliated protists** do not have IFT components **and**, like **some metazoan spermatozoa, use IFT-independent mechanisms to build axonemes exposed to the cytoso**l. Moreover, various ciliated protists lack TZ components, whereas Drosophila sperm surprisingly requires the activity of dynamically localized TZ proteins for cytosolic ciliogenesis. Here, we discuss the various ways eukaryotes use IFT and/or TZ proteins to generate the wide assortment of compartmentalized and cytosolic cilia observed in nature. Consideration of the different ciliogenesis pathways allows us to propose how three types of cytosol-exposed cilia (primary, secondary and tertiary), including cilia found in the human sperm proximal segment, are likely generated by evolutionary derivations of compartmentalized ciliogenesis.

My understanding is that in mammals, where most cells have a single "primary" non-motile cilium and various kinds of epithelial cells have multiple motile cilia, all of this ciliogenesis is compartmentalized. I'll get to sperm ciliogenesis in a bit.

From the paper on Twitchy you are asking about it clearly also occurs in Drosophila sperm ciliogenesis. With respect to annotation of the Twitchy paper, I think that since Drosophila has such a limited set of cells that make cilia at all, I think you could safely annotate to the "non-motile cilium assembly" term.

My recollection is "cytosolic ciliogenesis" is a bit of a special situation that occurs is certain places such as some of the proteozoa, which I think may be the reason @paolaroncaglia originally created the term.

The mammalian sperm ciliogenesis that I'm familiar with has extra assembly mechanisms like the manchette so I'm not completely sure how that would be categorized. I don't think I've ever encountered the term "cytosolic ciliogenesis" when reading about sperm ciliogenesis in mice. However this quote from the Twitchy paper you asked about as well as the Avidor-Reiss & Leroux abstract both talk about at least some metazoan spermatogenesis being cytosolic.

Hodge SH, et al. **Twitchy, the Drosophila orthologue of the ciliary gating protein FBF1/dyf-19, is required for coordinated locomotion and male fertility.** Biol Open. 2021 Aug 15;10(8):bio058531. doi: 10.1242/bio.058531. Epub 2021 Aug 6. PMID:34357392

> Primary cilia are compartmentalised from the rest of the cell by a ciliary gate comprising transition fibres and a transition zone. The ciliary gate allows the selective import and export of molecules such as transmembrane receptors and transport proteins. These are required for the assembly of the cilium, its function as a sensory and signalling centre and to maintain its distinctive composition. **Certain motile cilia can also form within the cytosol as exemplified by human and Drosophila sperm.** The role of transition fibre proteins has not been well described in the cytoplasmic cilia.

I wonder if we need to look more closely into whether sperm ciliogenesis and possible create terms for both cytosolic & compartmentalized versions..., noting at the same time that sperm structure is relatively varied across species/taxa and assembly might also be variable. I don't really have time to look into that right now though...

Does this address your current annotation needs @hattrill ?

If you think we should have new terms to address that at least some spermatogenesis is cytosolic, I guess we could leave the ticket open until I have a chance to consider if we should have multiple types of "sperm flagellum assembly", one term under "cytosolic ciliogenesis" & another under a term for "compartmentalized ciliogenesis" that does not yet exist.

Let me know what you think @hattrill

**Comment by paolaroncaglia** [comment]
Hi @krchristie and @hattrill ,

> My recollection is "cytosolic ciliogenesis" is a bit of a special situation that occurs is certain places such as some of the proteozoa, which I think may be the reason @paolaroncaglia originally created the term.

It was actually added by @ukemi in Jan 2017. There's no PMID. The only direct annotation is to Drosophila. 

Best,
Paola

**Comment by hattrill** [comment]
Hi @krchristie 
Thanks for that - it explains a lot and is very helpful. From what you've said, I think that I am happy to just explicity label when it is 'cytosolic ciliogenesis' rather than go down the rabbit hole of creating many more terms - a lot of the various flavours of ciliogenesis are just variations on a theme and are not when defined in how exactly they differ as far as I can see. I'll close this - it will be useful for future reference!

Hi @paolaroncaglia thanks for responding  - lovely to hear from you. Hope all is well with you!

**Comment by krchristie** [comment]
> Hi @krchristie
> Thanks for that - it explains a lot and is very helpful. From what you've said, I think that I am happy to just explicity label when it is 'cytosolic ciliogenesis' rather than go down the rabbit hole of creating many more terms 

works for me

>  a lot of the various flavours of ciliogenesis are just variations on a theme and are not when defined in how exactly they differ as far as I can see. 

agreed!!

> I'll close this - it will be useful for future reference!

Cool, I'm glad it's useful to you :)

**Comment by krchristie** [comment]
Hi @hattrill - I've reopened this because I read a little bit more of the Avidor-Reiss & Leroux mini-review of cytosolic vs compartmentalized ciliogenesis and I think that this does actually reflect some different mechanisms of ciliogenesis that merit better incorporation into the ontology. 

I have other higher priority projects right now, but I want to remember to get back to this. I'll definitely check in with you before making any changes that might affect Drosophila annotations.

**Comment by krchristie** [comment]
@hattrill - Now that I understand cytosolic ciliogenesis better, I wanted to add this quote from the Avidor-Reiss & Leroux mini-review since it might affect your understanding of how you want to do your annotations.

Basically, at least in sperm (I'm not as clear on the protozoa yet), "cytosolic ciliogenesis" is **NOT** a **_type_** of ciliogenesis, it is a **_part of_** of a specialized ciliogenesis that occurs in certain cells,  i.e. some sperm and some protozoa **AND** the compartmentalized portion of ciliogenesis occurs **_BEFORE_** the cytosolic phase of ciliogenesis in both mammalian and Drosophila sperm. 

So, when the paper you asked about says "Twitchy is required for compartmentalised and cytosolic ciliogenesis", you might want to look at whether they think Twitchy is involved in both compartmentalised and cytosolic ciliogenesis, OR whether Twitchy in ONLY involved in compartmentalized ciliogenesis, but thus disrupts cytosolic ciliogenesis as well because the preceding compartmentalized phase of ciliogenesis did not occur correctly.

> Drosophila Sperm Flagella


> Drosophila and other insects have compartmentalized sensory cilia and largely cytosolic sperm flagella. Like mammalian sperm and Giardia flagella, Drosophila spermatid flagella comprise two regions, but in Drosophila the proximal cytosolic axoneme is very long ($1,800 mm), whereas the distal compartmentalized cilium is very short ($2 mm) (Figure 3Av) [32,46].

> In spermatocytes, flagella formation initiates with compartmentalized ciliogenesis, whereby a basal body docks at the plasma membrane and forms a short ($1 mm) primary cilium (Figure 3Av). However, this compartmentalized ciliogenesis is independent of IFT, and the cilium is composed mainly of a TZ [21,24]. Later during meiosis, the basal body, together with the membrane-enclosed axoneme, invaginates into the cell and attaches to the nucleus. Subsequently, in spermatids, axonemal microtubules extend from the short compartmentalized region derived from the spermatocyte primary cilium. While this extension takes place, the TZ migrates distally along the axoneme and away from the basal body, leaving behind a cytosolic microtubule-based axoneme. The cytosolic portion elongates and ultimately forms the strikingly long cytosolic cilium. Cytosolic ciliogenesis continues by addition of dynein arms and other motile axoneme components directly from the cytoplasm [32,46]. Finally, the cytosolic axoneme associates with the plasma membrane through a process called individualization (reviewed in [47]).

> Several studies strongly argue that IFT is dispensable for Drosophila sperm cytosolic ciliogenesis. Mutations in kinesin-II or core IFT subcomplex components (such as nompB/IFT88/ Polaris/OSM-5, Oseg1/IFT122, and Oseg2/IFT172) disrupt sensory cilia, but not sperm formation [3,29,30]. In contrast, the TZ was recently shown to play a role in separating the cytosolic sperm region from the compartmentalized flagellar tip. Specifically, several TZ proteins, namely Cep290,
B9d1, B9d2, and Mks1, migrate away from the basal body to expose the proximal sperm axoneme to the cytoplasm [21]. The TZ protein Cep290 is essential for compartmentalization and axoneme assembly of the flagellar tip [21]. The axoneme forming at the flagellar tip later becomes the cytosolic axoneme and therefore disruption of Cep290 impacts the cytosolic axoneme.

> **Because the formation of a short compartmentalized cilium that depends on TZ proteins precedes the formation of the cytosolic axoneme, the TZ machinery may be indirectly essential for cytosolic ciliogenesis in Drosophila sperm.**

**Comment by hattrill** [comment]
Thanks @krchristie - that's  really helpful. I will go back and have another look with this in mind.

**Comment by pgaudet** [comment]
@hattrill What is the action for this one?

**Comment by raymond91125** [comment]
It appears that this was an issue about annotation choice and is resolved.
